README.md

```diff
@@ -6,42 +6,45 @@ Golang SDK for interacting with the Wavefront v2 API, and sending metrics throug
 
 ### API Client
 
-Presently support for:
- * Querying
- * Searching
- * Dashboard Management
- * Alert (and Alert Target) Management
- * Events Management
-
-Please see the [examples](examples) directory for an example on how to use each, or check out the [documentation](https://godoc.org/github.com/WavefrontHQ/go-wavefront-management-api).
+Supports most public API paths. Refer to the
+[documentation](https://godoc.org/github.com/WavefrontHQ/go-wavefront-management-api)
```

Comment:
How would you feel about linking to pkg.go.dev/github.com/WavefrontHQ/go-wavefront-management-api instead?  pkg.go.dev is module aware and we use go modules.  godoc.org just shows the latest API checked into github, but pkg.go.dev shows the latest release which could be different. Sometimes we check changes into github, but release those changes much later.